@types/Polyline.d.ts

```diff
@@ -65,6 +65,11 @@ declare namespace kakao.maps {
   export type setPolylineOptions = Partial<PolylineOptions>
 
   export interface PolylineOptions {
+    /**
+     * 폴리라인이 올라갈 지도 또는 로드뷰
+     */
+    map?: Map | Roadview | Roadview;
```

Comment:
테스트를 해보니 `Map`를 받고 있는 것을 확인했습니다.! 문서상에도 `Map`를 받는 형태로 예시가 작성되어 있지만, Options에 포함이 안되어 있어서 타입 정의를 못한 것 같습니다.
추가로 `Polyline` 이외에 `Circle`, `Rectangle`, `Polygon` 등 다양한 도형 타입에도 추가가 가능한 것으로 확인되었습니다.!

다만, `Roadview`의 경우 실제로 테스트 해본 결과 동작하지 않고, 문서 상에도 존재하지 않아 해당 부분은 수정되어야 할 것 같습니다. (+ 추가로 `Roadview | Roadview` 중복으로 작성하신 부분 때문에 만약 해당 옵션을 받더라도 수정이 이루어져야 할 것 같습니다 ㅠㅠ)

기여 감사합니다.!

```suggestion
    /**
     * 폴리라인이 올라갈 지도
     */
    map?: Map;
```

Reply:
엇! 제가 로드뷰의 정의를 헷갈려서 두번 작업하게 만들었네요. 죄송합니다!
circle과 rectangle, polygon에서도 추가 가능하다면 함께 추가해 올리겠습니다!
빠른 리뷰 감사합니다!